### PR TITLE
Fixed: (Orpheus) Add remaster title and year to release title

### PR DESCRIPTION
**Pull request by mynameisbogdan**
#### Database Migration
NO

### CODE REVIEW

**Comment by bakerboy448** [comment]
gotta update the orpheus tests too ;)

```
"The Beatles - Abbey Road (1969) [MP3 V2 (VBR)] [BD]" with a length of 51, but
"The Beatles - Abbey Road (1969) [2.0 Mix 2019] [MP3 V2 (VBR)] [BD]" has a length of 66, differs near "2.0" (index 33).
```

**Comment by Qstick** [comment]
Wonder how well this format plays with Lidarr.. might need some tweaks to that parser.

File: src/NzbDrone.Core/Indexers/Definitions/Orpheus.cs [hunk 1]
```diff
@@ -314,7 +299,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     var id = result.TorrentId;
                     var infoUrl = GetInfoUrl(result.GroupId, id);
 
-                    GazelleInfo release = new GazelleInfo()
+                    GazelleInfo release = new GazelleInfo
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Definitions/Orpheus.cs:302]: Use `var` here as well like you did above?

**Review by Qstick — CHANGES_REQUESTED** [review]
Just the one nitpick, looks ok otherwise.

**Comment by bakerboy448** [comment]
imageimage

**Review by Qstick — APPROVED** [review]

**Pull request merged**